testsuite/features/support/env.rb

```diff
@@ -192,7 +201,13 @@ def process_code_coverage
 end
 
 AfterStep do
-  log 'Timeout: Waiting AJAX transition' if has_css?('.senna-loading', wait: 0) && !has_no_css?('.senna-loading', wait: 30)
+  next unless Capybara::Session.instance_created?
```

Comment:
This change doesn't look related to the new changes.

Reply:
That's a leftover of the bigger refactor I made, to fully re-enable parallel tests. This was necessary to recover properly when we run tests using parallel_tests gem, and some scenario fails.

I can revert it, and keep in another branch for now, still we gonna need it.